### DISCUSSION TITLE
Kimai problems after update of mysql

### DISCUSSION THREAD

**Discussion by drwemeier** [kimai/kimai#3150] [On-Premise: Installation & Updates]
Our KIMAI-Instance is hosted by host europe and ran smoothly since first installation in December 2021. Beginning at Tuesday last, we're facing unpredictable errors that have no corresponding entries in the log. The error is either a message indicating a gateway error (502) or the page is not produced. In the latter case a 404-page is displayed by the hoster.
The only difference we could identify is an updated version of mysql. The error reoccurs with the adjusted connection string in .env
![mysql](https://user-images.githubusercontent.com/99537725/153713320-2d964b78-bb84-4e60-becc-92380ebdaf63.png)

![env](https://user-images.githubusercontent.com/99537725/153713359-eb07b0cd-0120-4e38-96a1-935148076e00.png)

Reloading KIMAI by bin/console kimai:reload solves the problem for a limited period of time.
We ran composer and bin/console kimai:update on the current installation with no effect.
We're running KIMAI 1.16.5

Can anybody help us with the problem

**Comment by kevinpapst** [comment]
First of all: if nothing changed, except the database, then it is likely the database/your hoster is the one to blame and not the software only accessing it. 

That being said I doubt that the connection string has anything to do with the problem, but you should still make it `serverVersion=5.7.36`

Upgrade to the latest Kimai release. If the error persists, ask Hosteuope.
